manifests/install.pp

```diff
@@ -48,7 +48,7 @@
         }
 
         case $::lsbdistcodename {
```

Comment:
The current way to call facts is to use `$facts` and with structured facts.
Can you update with `$facts['os']['distro']['codename']` ?

And update `metadata.json` accordingly. It is used by tools to know compatibility of module.

Reply:
Not related to this code, but just wanted to mention that if there's a risk that some nodes may not have a particular fact one can use [built-in function `dig`](https://puppet.com/docs/puppet/latest/function.html#dig); e.g.

```puppet
$a = $facts.dig('some', 'weird', 'fact') # $a might be undef
$b = $facts.dig('some', 'weird', 'fact').lest || {
  'a terrible default value'
}   # if dig ends up returning an undef, then the value of $b will be the string
    # 'a terrible default value'
```